querydsl-sql/src/test/java/com/querydsl/sql/SelectBase.java

```diff
@@ -395,7 +395,7 @@ public void Custom_Projection() {
     }
 
     @Test
-    @IncludeIn({H2, SQLSERVER, MYSQL, ORACLE, TERADATA}) // TODO fix postgresql
+    @ExcludeIn({CUBRID, DB2, DERBY, HSQLDB, POSTGRESQL, SQLITE, TERADATA})
```

Comment:
I suggest that we also apply this on the other branch, so we don't pull our hairs out every time we change this line.
We could also make it a separate PR to equalize this everywhere.